source/localizable/routing/defining-your-routes.md

```diff
@@ -29,6 +29,16 @@ Router.map(function() {
 });
 ```
 
+If your route name has multiple works then your route should follow convention and be hyphenated. Routes with underscores in them will not be rendered.
```

Comment:
It's hard to explain in a way that doesn't talk about the resolver/container, which we want to avoid :P but the shorter explanation is that all of the classes registered with Ember are dasherized, so that's how it expects the lookups to be. `path` is what shows up in the URL, which can be anything (as long as it's a valid URL :P).